main.tf

```
@@ -20,7 +20,7 @@ locals {
   bucket_encryption_enabled           = var.bucket_encryption_enabled && length(local.bucket_sse_key_arn) > 0
   bucket_sse_key_arn                  = var.bucket_encryption_enabled ? (length(var.bucket_sse_key_arn) > 0 ? var.bucket_sse_key_arn : aws_kms_key.lacework_eks_kms_key[0].arn) : ""
   sns_topic_key_arn                   = var.sns_topic_encryption_enabled ? (length(var.sns_topic_encryption_key_arn) > 0 ? var.sns_topic_encryption_key_arn : aws_kms_key.lacework_eks_kms_key[0].arn) : ""
-  kinesis_firehose_encryption_enabled = var.kinesis_firehose_encryption_enabled && length(var.kinesis_firehose_encryption_key_arn) > 0
+  kinesis_firehose_encryption_enabled = var.kinesis_firehose_encryption_enabled && length(local.kinesis_firehose_encryption_key_arn) > 0
```

Is the name of the local variable correct?
```suggestion
  kinesis_firehose_encryption_enabled = var.kinesis_firehose_encryption_enabled && length(local.kinesis_firehose_key_arn) > 0
```

---

You're right. That has "encryption" in it and the local variable doesn't. Let me close this PR and actually submit a new one with the right local var name. I am also going to rename other variables for consistency purposes.